sdk/examples/intkey_python/sawtooth_intkey/client_cli/intkey_workload.py

```diff
@@ -133,15 +133,17 @@ def on_batch_committed(self, batch_id):
 
                 batch_list = batch_pb2.BatchList(batches=[batch])
 
-                post_batches(key.url, batch_list, auth_info=self._auth_info)
-
-                with self._lock:
-                    self._pending_batches[batch.header_signature] = \
-                        IntKeyState(
-                        name=key.name,
-                        url=key.url,
-                        value=key.value + 1)
-                self.delegate.on_new_batch(batch_id, key.url)
+                (code, _) = post_batches(key.url, batch_list,
+                                         auth_info=self._auth_info)
+
+                if code == 202:
+                    with self._lock:
+                        self._pending_batches[batch.header_signature] = \
+                            IntKeyState(
+                            name=key.name,
+                            url=key.url,
+                            value=key.value + 1)
+                    self.delegate.on_new_batch(batch_id, key.url)
```

Comment:
Do we want any visibility on when a batch is rejected? In this case, is the batch just dropped?

Reply:
We do log a message on the failure with the error code and status message, in the call to post batches.  We could add one that logs something more specific to the 429 error that results here.  

This fix also corrects a case where the rest api is returning any non-202 status - it previously was assuming that all submits resulted in a successful submission (regardless if the batch is committed).